cmd/controller/main.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	// The set of controllers this controller process runs.
+	"knative.dev/net-gateway-api/pkg/reconciler/ingress"
+
+	// This defines the shared main for injected controllers.
+	"knative.dev/pkg/injection/sharedmain"
+)
+
+func main() {
+	sharedmain.Main("controller",
```

Comment:
Can we use a more specific name instead of "controller"? e.g. net-gateway-api-controller

https://github.com/knative-sandbox/net-istio/blob/main/cmd/controller/main.go#L34